cmd/kubeadm/app/phases/selfhosting/selfhosting.go

```diff
@@ -49,26 +48,24 @@ const (
 // It achieves that task this way:
 // 1. Load the Static Pod specification from disk (from /etc/kubernetes/manifests)
 // 2. Extract the PodSpec from that Static Pod specification
-// 3. Mutate the PodSpec to be compatible with self-hosting (add the right labels, taints, etc. so it can schedule correctly)
+// 3. Mutate the PodSpec to be compatible with selfhosting (add the right labels, taints, etc. so it can schedule correctly)
```

Comment:
**we are not consistent with naming of the feature.**

please rename all comments and user outputs (printf, println...) to be `self-hosting` as per the wikipedia definition:
https://en.wikipedia.org/wiki/Self-hosting

Reply:
do we want the command to be `self-hosting` then as well?

Reply:
please keep the filename selfhosting.go but the command and println() + comment should be self-hosting.